### ISSUE TITLE
Unable to change color of model for stp file

### ISSUE THREAD

**Issue by chriskuech** [kovacsv/Online3DViewer#332]
**Describe the bug**
I changed the `defaultColor` everywhere I see it hard-coded (`settings.js`, `sidebarsettings.js`). However, the model color for our stp file is still the default `rgb(59, 59, 59)`/`#3B3B3B`. Possibly related, there is no "Default Color" picker UI, unlike what's shown in #264 screenshot.

**To Reproduce**
1. Change the `defaultColor` in all cases within the website code.
1. Upload an stp file.

**Expected behavior**
Color of the model is the `defaultColor`.

**Screenshots**
<img width="816" alt="image" src="https://user-images.githubusercontent.com/3680671/206011253-6e0dee6c-bf11-49d6-bb5c-c777fa1b7474.png">

**Comment by kovacsv** [comment]
Default material has effect only if the model doesn't specify the color (like in case of the stl format). In this case your file has a color specified, so default color doesn't make sense, this is why it doesn't appear on the UI.

**Comment by chriskuech** [comment]
I see. Makes sense, but it would be nice to change the color of the model regardless.

**Comment by clintmorris229** [comment]
I agree that  it would be nice to change the model color. Especially when facets are colored the same color as the default background.

**Comment by barbinbrad** [comment]
Agree with the above sentiment. I'd like to be able to change the color to reflect the material/finish of the part. Any workarounds for this?
